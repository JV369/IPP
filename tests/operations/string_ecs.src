.IPPcode18
write string@hello\032world\010ahoj\032svete!
write string@\104\101\108\108\111\032\119\111\114\108\100\010\097\104\111\106\032\115\118\101\116\101\033