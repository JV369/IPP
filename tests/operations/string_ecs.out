hello world
ahoj svete!
hello world
ahoj svete!
